deployments/apiv2/services/users/src/main.py

```diff
@@ -236,7 +245,7 @@ async def logout(request: Request, token=Depends(ProtectedAny(check_scope=False)
 async def register(
     request: Request,
     details: Union[CustomerCreate, UserCreate],
-    token=Depends(ProtectedAny(scope=["users:admin"])),
+    token=Depends(ProtectedAny(scope=PULSE3D_CUSTOMER_SCOPES)),
```

Comment:
I don't think this route should be specific to pulse3d users, the customer admin account won't be tied to a specific product